tests/config/test_override_config.py

```diff
@@ -88,12 +88,12 @@ def test_override_default_config_with_file_config_including_env_variable_values(
     with mock.patch.dict(os.environ, {"FOO": "foo", "BAR": "true"}):
         with pytest.raises(InconsistentEnvVariableError):
             Config.load(tf.filename)
-            Config.global_config.foo_attribute
+            _ = Config.global_config.foo_attribute
```

Comment:
It looks strange to me.

Reply:
These are because `Config.global_config.foo_attribute` is considered as useless because the retrieved value is not assigned to anywhere